src/commands/mod.rs

```diff
@@ -136,6 +138,8 @@ impl fmt::Display for CommandError {
                 f,
                 "No coin currently spendable through this timelocked recovery path."
            ),
+            Self::InvalidAddressCount => write!(f, "Invalid address count, should be under 2^31-1"),
+            Self::InvalidAddressIndex => write!(f, "Invalid address index, should be under 2^31-1"),
```

Comment:
Would be better to introduce a single enum variant, a general "invalid derivation index" which could be reused in the future.